eng/pipelines/templates/build-pull-request.yml

```diff
@@ -78,7 +78,9 @@ jobs:
 
   # Publishes the artifacts folder contents to the Azure Pipeline for use in other jobs/stages.
   # This publish is filtered by the .artifactignore in the 'eng' folder. See that file for more details.
-  - publish: $(Build.SourcesDirectory)/artifacts/$(BuildConfiguration)
+  - task: 1ES.PublishPipelineArtifact@1
```

Comment:
1ES shouldn't appear in PR builds

Reply:
I don't believe that's correct. See https://dev.azure.com/devdiv/DevDiv/_wiki/wikis/DevDiv.wiki/38967/Guidance-for-PR's-and-CI's-PLEASE-READ - 1ES checks can occur during PR builds.